WordPress/gutenberg
# [RNMobile] Bump `react-native-gesture-handler` to version `2.3.2`

## What?


Bumps the package `react-native-gesture-handler` to version `2.3.2`.

## Why?


Starting in version `2.3.0`, the library provides APIs that allow testing gestures. In an incoming PR, we're introducing integration tests related to the drag & drop that requires using these APIs.

## How?


This PR updates the reference to the library in the different files where it's referenced.

## Testing Instructions




1. Run the Gutenberg demo app and observe that works as expected.
2. Observe that drag & drop functionality works as expected.

Additionally, it would be great to test this change in the main apps:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/18742
- https://github.com/wordpress-mobile/WordPress-Android/pull/16645

## Screenshots or screencast 
N/A
